tests/install/openqa_worker.pm

```diff
@@ -5,7 +5,7 @@ use utils;
 sub run {
     diag('worker setup');
     disable_packagekit;
-    assert_script_run('retry -e -s 30 -r 7 -- sh -c "zypper -n --gpg-auto-import-keys ref && zypper --no-cd -n in openQA-worker"', 3800);
+    assert_script_run('retry -e -s 30 -r 7 -- sh -c "zypper -n --gpg-auto-import-keys ref && zypper --no-cd -n in openQA-worker os-autoinst-openvswitch"', 3800);
```

Comment:
Why is it necessary to pull in another package? We should make sure that our packages as well as documentation bring along everything that is necessary to run multi-machine tests. Actually os-autoinst-setup-multi-machine according to our docs pull that in so that should be used instead